src/new-apps/components/AppListCard/AppListCardDescription.tsx

```diff
@@ -1,44 +1,54 @@
 import { GetV2SaleorAppsResponse } from "@dashboard/new-apps/marketplace.types";
-import { Typography } from "@material-ui/core";
+import { Box, Text } from "@saleor/macaw-ui/next";
 import React from "react";
 
-import { useDescriptionStyles } from "./styles";
-
 interface AppListCardDescriptionProps {
   app: GetV2SaleorAppsResponse.SaleorApp;
 }
 
 const AppListCardDescription: React.FC<AppListCardDescriptionProps> = ({
   app,
-}) => {
-  const classes = useDescriptionStyles();
-
-  return (
-    <>
-      <div className={classes.cardToolbar}>
-        <div
-          className={classes.logo}
-          style={{
-            backgroundColor: app.logo.color,
-          }}
-          data-test-id="app-logo"
-        >
-          {app.logo.source && <img src={app.logo.source} alt="App logo" />}
-          {!app.logo.source && (
-            <Typography variant="h2" data-test-id="app-logo-placeholder">
-              {app.name.en[0] || ""}
-            </Typography>
-          )}
-        </div>
-        <Typography className={classes.cardHeader} variant="h2">
-          {app.name.en}
-        </Typography>
-      </div>
-      <Typography className={classes.description} variant="body1">
-        {app.description.en}
-      </Typography>
-    </>
-  );
-};
+}) => (
+  <Box marginBottom={6}>
+    <Box
+      display="flex"
+      flexDirection="row"
+      alignItems="center"
+      marginBottom={8}
+      gap={6}
+    >
+      <Box
+        width={13}
+        height={13}
+        display="flex"
+        placeItems="center"
+        borderRadius={3}
+        style={{
+          backgroundColor: app.logo.color,
+        }}
+        data-test-id="app-logo"
+      >
+        {app.logo.source && <img src={app.logo.source} alt="App logo" />}
+        {!app.logo.source && (
+          <Text
+            variant="bodyEmp"
+            size="large"
+            as="h2"
+            data-test-id="app-logo-placeholder"
+            color="textNeutralContrasted"
+          >
+            {app.name.en[0] || ""}
+          </Text>
+        )}
+      </Box>
```

Comment:
question: can't you use AppAvatar here ?

Reply:
Designs are still being discussed - if we use placeholder icon then yes, if we use initials then not.